src/main/java/service/strategy/strategyimpl/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package service.strategy.strategyimpl;
+
+import dao.Storage;
+import model.FruitRecord;
+
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void apply(FruitRecord transaction) {
```

Comment:
Please follow the encapsulation principle. It's better to have class elements hidden with the help of access modifiers. In this case, the class `ReturnOperation` should not be public if it is not used outside of its package.